keps/sig-node/127-user-namespaces/kep.yaml

```diff
@@ -15,7 +16,7 @@ approvers:
   - "@derekwaynecarr"
 
 stage: alpha
-latest-milestone: "v1.27"
+latest-milestone: "v1.28"
 milestone:
   alpha: "v1.25"
```

Comment:
Can you pls update it to beta?

Reply:
Yes, but then I would have to incorporate the whole PRR. Should we do that now or as follow-up?